helm/kubernetes-cluster-prep/bin/get-conjur-cert.sh

```diff
@@ -205,14 +214,16 @@ function k8s_verify_certificate() {
   cert_filepath="$2"
   conjur_url="$3"
```

Comment:
We should change this passed argument from `conjur_url` to `domain_name`. You'd also have to change where `k8s_verify_certificate` gets invoked earlier to actually pass the `domain_name` variable, but that variable is available at that invocation.

This would simplify and make things clearer in that you'd be able to skip the next line where you call `get_domain_name`, and use the domain name directly, e.g.:
```
  curl_cmd="curl --cacert $cert_filename $domain_name"
```

Or if you'd prefer, change the next line to:
```
  url="https://$domain_name"
```

Reply:
yes, that is cleaner